### ISSUE TITLE
Regression: Filter keyboard navigation

### ISSUE THREAD

**Issue by lordievader** [vector-im/element-web#11314]
![image](https://user-images.githubusercontent.com/3618054/68193098-bf48ed80-ffb2-11e9-8b84-c64fdd3530e8.png)

### Description
I really like keyboard driven approaches. What I typically do to switch channels is hit Ctrl+K to enter the filter dialog. Search for the channel I want and use the arrow keys to select the channel to view.
Somewhere between v1.5.0 and v1.5.1 the up arrow key broke. As shown in the screenshot, if you want to go to 'IRC Matrix Bridges' quickly I used to hit the up arrow once and end up highlighting the correct room. Now I have to hit the down arrow a bunch of times to arrive at the same.

### Steps to reproduce
- Hit Ctrl-K to enter the filter dialog
- Search for something
- Hit up arrow to select the room listed at the bottom
- Notice it doesn't work

In v1.5.0 it would just wrap around and jump to the room listed at the bottom.

### Version information

Version with the problem is v1.5.2.

- **Platform**: desktop (pulled from the [Debian repo][1] packages)

For the desktop app:

- **OS**: Gentoo
- **Version**: Very recent (rolling release)

[1]: https://packages.riot.im/debian/pool/main/r/riot-web/

**Comment by t3chguy** [comment]
Sadly this is not a regression and is actually done to match an ARIA authoring practice pattern known as Navigation Treeview: https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html

**Comment by t3chguy** [comment]
@pvagner how unclear would things be for you if arrow keys wrapped top/bottom on this widget?

**Comment by pvagner** [comment]
@t3chguy If it's doable I don't mind if this would wrap. It should be consistent i.e. when pressing up arrow on the first item usually people item, it should jump to the very last item. And the other way round. Pressing down arrow on the last item should move to the first item.

**Comment by t3chguy** [comment]
I will make a PoC with this soon and have product team play with it

**Comment by t3chguy** [comment]
PoC build in https://riots.im/adhoc/a11y_poc/

**Comment by lordievader** [comment]
In a very quick test this seems to do exactly what I had in mind :+1:

**Comment by t3chguy** [comment]
The biggest difference in what Tab does as opposed to what Arrow Keys do

**Comment by jryans** [comment]
@t3chguy Is this valid with the new room list...?

**Comment by t3chguy** [comment]
Indeed, it stops at top and bottom and doesn't loop around, whether or not it should is unclear.

**Comment by t3chguy** [comment]
This interaction is gone
